src/pages/components/tile/usage.mdx

```diff
@@ -38,10 +40,18 @@ the issues.
 
 Carbon ships a basic tile structure that responds to the grid. Based on the
 layout structure, tiles can contain type, images and/or a block of color. As
-tiles have no pre-set styles for the content within them, the tile component
-usage guidance is purposefully high-level. It focuses specifically on the tile
-itself, not the structure of the information or interactive elements that the
-tile contains—that type of information will be found in the card pattern.
+tiles have no pre-set styles for the content inside, the tile component usage
+guidance focuses on basic tiles' functions. It does not include the complex
+structure of the information or additional interactive elements that a tile
+contains.
+
+<Row>
```

Comment:
I am not sure why this image is not showing up. Maybe @francinelucca can help.

Reply:
@tay1orjones can you run the vercel deployment with a clean cache ? I don't think I have access

Reply:
can you see it now @laurenmrice ? https://carbondesignsystem-git-fork-thyhmdo-757abb-carbon-design-system.vercel.app/components/tile/usage/

Reply:
Yes! Thank you for fixing it!